### ISSUE TITLE
Model Overfitting at less time.

### ISSUE THREAD

**Issue by SanthoshRajendiran** [PINTO0309/TensorflowLite-UNet#2]
First of all.. Congrats on the awesome work done. We tried out your code.. We could infer that, the model starts over-fitting in very few epochs itself. We could not imagine model to learn those many features in a short span of time. When we tried to do more after around 60 epochs, the test loss starts increasing. Could you suggest any way to improve the final model accuracy? Though we have huge datasets like Supervisely and PFCN, this problem still exist. Any suggestions would be helpful..
